service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementSubmittedControllerTest.java

```diff
@@ -0,0 +1,131 @@
+            .caseDetails(CaseDetails.builder()
+                .id(CASE_ID)
+                .data(ImmutableMap.<String, Object>builder()
+                    .putAll(buildNotificationData())
+                    .putAll(buildPlacementData(List.of(child1, child2), List.of(child2Placement, child1Placement),
+                        child2.getId()))
+                    .build())
+                .build())
+            .caseDetailsBefore(CaseDetails.builder().data(new HashMap<>()).build())
+            .build();
+
+        postSubmittedEvent(callbackRequest);
+
+        verify(notificationClient).sendEmail(
+            eq(PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE), eq("admin@family-court.com"),
+            eq(expectedTemplateParameters()), eq(CASE_REFERENCE));
+    }
+
+    @Test
+    void shouldNotSendANotificationWhenUploadingPreviousChildPlacement() throws Exception {
+        Element<Child> child1 = testChild();
+        DocumentReference child1Application = testDocument();
+        Element<Placement> child1Placement = element(testPlacement(child1, child1Application));
+
+        CallbackRequest callbackRequest = CallbackRequest.builder()
+            .caseDetails(CaseDetails.builder()
+                .data(buildPlacementData(List.of(child1), List.of(child1Placement), child1.getId()))
+                .build())
+            .caseDetailsBefore(CaseDetails.builder()
+                .data(buildPlacementData(List.of(child1), List.of(child1Placement), child1.getId()))
+                .build())
+            .build();
+
+        postSubmittedEvent(callbackRequest);
+
+        verify(notificationClient, never()).sendEmail(
+            eq(PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE), eq("admin@family-court.com"),
+            eq(expectedTemplateParameters()), eq(CASE_REFERENCE));
+    }
+
+    private Map<String, Object> buildPlacementData(List<Element<Child>> children,
+                                                  List<Element<Placement>> placements,
+                                                  UUID childID) {
+        return Map.of(
+            "children1", children,
+            "placements", placements,
+            "childrenList", childID);
+    }
+
+    private Map<String, Object> expectedTemplateParameters() {
+        return ImmutableMap.of(
+            "respondentLastName", RESPONDENT_SURNAME,
+            "caseUrl", "http://fake-url/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345"
+        );
+    }
+
+    private Map<String, Object> buildNotificationData() {
+        return ImmutableMap.of(
+            "caseLocalAuthority", LOCAL_AUTHORITY_CODE,
+            "respondents1", List.of(
+                ImmutableMap.of(
+                    "value", Respondent.builder()
+                        .party(RespondentParty.builder()
+                            .lastName(RESPONDENT_SURNAME)
+                            .build())
+                        .build()))
```

Comment:
any value just having 
```java
wrapElements(Respondent.builder()
                .party(RespondentParty.builder()
                    .lastName(RESPONDENT_SURNAME)
                    .build()))
```
in place of 
```java
List.of(
                ImmutableMap.of(
                    "value", Respondent.builder()
                        .party(RespondentParty.builder()
                            .lastName(RESPONDENT_SURNAME)
                            .build())
                        .build()))
```
?

Reply:
It makes it more readable in my opinion so yes it does add value.

Reply:
Not particularly, i'll keep as it is if you don't mind.